Issue {issueClosedAt = Nothing, issueUpdatedAt = GithubDate {fromGithubDate = 2013-03-27 16:13:25 UTC}, issueHtmlUrl = "https://github.com/fail2ban/fail2ban/issues/161", issueClosedBy = Nothing, issueLabels = [IssueLabel {labelColor = "02d7e1", labelUrl = "https://api.github.com/repos/fail2ban/fail2ban/labels/moreinfo", labelName = "moreinfo"}], issueNumber = 161, issueAssignee = Nothing, issueUser = GithubUser {githubOwnerAvatarUrl = "https://secure.gravatar.com/avatar/6a424256589825f609fd3477144355a0?d=https://a248.e.akamai.net/assets.github.com%2Fimages%2Fgravatars%2Fgravatar-user-420.png", githubOwnerLogin = "yarikoptic", githubOwnerUrl = "https://api.github.com/users/yarikoptic", githubOwnerId = 39889, githubOwnerGravatarId = "6a424256589825f609fd3477144355a0"}, issueTitle = "asyncore: (9, 'Bad file descriptor')", issuePullRequest = PullRequestReference {pullRequestReferenceHtmlUrl = Nothing, pullRequestReferencePatchUrl = Nothing, pullRequestReferenceDiffUrl = Nothing}, issueUrl = "https://api.github.com/repos/fail2ban/fail2ban/issues/161", issueCreatedAt = GithubDate {fromGithubDate = 2013-03-27 16:13:25 UTC}, issueBody = "```\r\nTesting using /usr/bin/python2.6\r\nFail2ban 0.8.8 test suite. Python 2.6.8 (unknown, Jan 26 2013, 14:35:25) [GCC 4.7.2]. Please wait...\r\n.........................................Exception in thread Thread-61:\r\nTraceback (most recent call last):\r\n  File \"/usr/lib/python2.6/threading.py\", line 532, in __bootstrap_inner\r\n    self.run()\r\n  File \"/usr/lib/python2.6/threading.py\", line 484, in run\r\n    self.__target(*self.__args, **self.__kwargs)\r\n  File \"/home/yoh/deb/gits/fail2ban/server/asyncserver.py\", line 144, in start\r\n    asyncore.loop(use_poll = False) # fixes the \"Unexpected communication problem\" issue on Python 2.6 and 3.0\r\n  File \"/usr/lib/python2.6/asyncore.py\", line 210, in loop\r\n    poll_fun(timeout, map)\r\n  File \"/usr/lib/python2.6/asyncore.py\", line 140, in poll\r\n    r, w, e = select.select(r, w, e, timeout)\r\nerror: (9, 'Bad file descriptor')\r\n\r\n.....................................F....\r\n```\r\n\r\nNever ran into this one before (or at least not for a while ;) )", issueState = "open", issueId = 12519498, issueComments = 0, issueMilestone = Nothing}